backend/sample_data/database_template.json5

```diff
@@ -54,13 +44,13 @@
             ],
             "description": "How much data do you need to store? Less than 1 million records is small. Within 1 million and 100 million is medium. And above 100 million records is large."
         },
-        "complex_query_patterns": {
+        "fast_response_time": {
             multi_choices: false,
             "accepted_values": [
                 true,
                 false
             ],
-            "description": "Do you have to do a lot of complex queries?"
+            "description": "Do your project require microsecond response time from database? For example, leaderboards or session store may requires it."
```

Comment:
tại sao leaderboards or session store lại require microsecond response time nhỉ?

Reply:
Cái này là vì use case của mấy thằng này yêu cầu cập nhật dữ liệu cũng như lấy dữ liệu rất nhanh.
Đây là use case của các Database chạy trực tiếp trên RAM (In-memory use case)

Reply:
cái này là thuộc kiểu "có thì càng tốt" nên lí do gì user sẽ không chọn true nhỉ?

Reply:
chắc là ở phần UI thêm chú ý kiểu "hãy chọn các điều kiện đúng với nhu cầu của bạn để có thể lựa chọn database tối ưu về chi phí nhất"

Reply:
If the analysis results don't meet your needs, adjust it to get the most cost-effective database.

Reply:
https://github.com/vinhyenvodoi98/LLM_hackathon_2023/issues/36
thêm issue ở đây